reporters/fail.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var PluginError = require('gulp-util').PluginError;
+var through = require('through2');
+
+module.exports = function (opts) {
+	opts = opts || {};
+
+	// paths to files that failed JSCS
+	var fails = false;
+
+	// files that need to be passed downstream on flush
+	var buffer = opts.buffer !== false ? [] : false;
```

Comment:
Oh, correction: it doesn't make much sense to use `gulp-plumber` with a `fail` reporter. The whole point of `buffer` is not breaking the pipe after some of the files went through the `fail` reporter thus preventing inconsistent build state.